examples/kaguya_isis_cmp.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing out a USGSCSM ISD from a PDS3 Kaguya image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pvl\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import knoten\n",
+    "import csmapi\n",
+    "\n",
+    "os.environ['ISISROOT'] = '/usgs/pkgs/isis3.8.0_RC1/install'\n",
+    "from pysis import isis\n",
+    "from pysis.exceptions import ProcessError"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Make a CSM sensor model\n",
+    "Requires TC1S2B0_01_02842S506E1942.img in data directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fileName = 'data/TC1S2B0_01_02842S506E1942.img'\n",
+    "camera = knoten.csm.create_csm(fileName)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the output location of the resulting .cub\n",
+    "cub_loc = 'data/TC1S2B0_01_02842S506E1942.cub'\n",
```

Comment:
Could we strip the path and the name of the cube (excluding the extension) from the image since the cube and the image will be named the same? This way, we don't have to update the path in multiple areas if we decide to change it.